development/setup-osx.rb

```diff
@@ -39,14 +55,25 @@ def run_install_script(program, noisy=true)
     when 'bower'
       system('npm install -g bower')
     when 'npm'
-      system('curl https://raw.githubusercontent.com/creationix/nvm/v0.24.1/install.sh | bash')
+      system('curl https://raw.githubusercontent.com/creationix/nvm/v0.24.1/install.sh | bash') # TODO find a way to figure out latest version
       system('nvm install stable')
     when 'ruby-build'
       system('brew install ruby-build')
     when 'rbenv'
       system('brew install rbenv')
+    when 'psql'
```

Comment:
This is fine to merge the PR, but we should move the conditionals into a command pattern and lookup the command based off of the application name. Maybe add an issue for this?